src/Tools/dotnet-monitor/Extensibility/ProgramExtension.cs

```diff
@@ -54,8 +54,6 @@ public async Task<EgressArtifactResult> EgressArtifact(ExtensionEgressPayload co
                 ExtensionException.ThrowWrongType(_extensionName, _declarationPath, typeof(IEgressExtension));
             }
 
-            // This _should_ only be used in this method, it can get moved to a constants class if that changes
-            const string CommandArgProviderName = "--Provider-Name";
```

Comment:
I know Justin made a comment on my initial PR about [this](https://github.com/dotnet/dotnet-monitor/pull/2748#discussion_r1002185585); IIRC, there was a discussion during one of my design reviews where we talked about removing this. It was originally included so that extensions could control their own configuration if they wanted to; however, to keep the experience consistent for users we weren't sure we wanted to encourage that. At this time, the `ProviderName` is still being included as part of `ExtensionEgressPayload` and I've only stripped it out as a command line arg; I'm open to suggestions on what course of action we should take in terms of leaving this in, or completely ripping it out.